src/main/java/seedu/address/logic/commands/vendor/AddVendorCommand.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.logic.commands.vendor;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEDDING;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a vendor to the address book.
+ */
+public class AddVendorCommand extends Command {
+
+    public static final String COMMAND_WORD = "add-vendor";
+
+    // to add task field prefix and example at the end
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds the person identified by the index number used in the last person listing "
```

Comment:
From what I see the add-vendor command seems to be adding a vendor by specifying the fields of a person to add instead of by the index used in the last person listing? If that's the case I think there may be a typo in this comment